source/guide_conduit.rst

```diff
@@ -0,0 +1,215 @@
+.. author:: Jonas <https://github.com/jfowl>
+
+.. tag:: lang-rust
+.. tag:: chat
+
+.. highlight:: console
+
+.. sidebar:: Logo
+
+  .. image:: _static/images/conduit.svg
+      :align: center
+
+#######
+Conduit
+#######
+
+.. tag_list::
+
+Conduit_ is a homeserver for the Matrix chat protocol written in :manual:`Rust <lang-rust>`. It is targeted at personal use.
+
+At the time of writing Conduit has not reached a stable 1.0 version yet.
+Until then, upgrading it might incur data loss.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * :manual:`supervisord <daemons-supervisord>`
+
+Installation
+============
+
+Download the latest development build and mark it as executable:
```

Comment:
You download the binary directly into the home folder, how about `~/bin`?

Reply:
Done.